bottles/backend/health.py

```diff
@@ -104,7 +104,7 @@ def check_x11(self):
 
     @staticmethod
     def check_wayland():
-        if "WAYLAND_DISPLAY" in os.environ:
+        if "WAYLAND_DISPLAY" in os.environ or "WAYLAND_SOCKET" in os.environ:
```

Comment:
```suggestion
        if "WAYLAND_DISPLAY" or "WAYLAND_SOCKET" in os.environ:
```
Would this not work?

Reply:
Negative, Python sees that as ```if True or x in y``` and would always resolve as True.